File: springboot/ps5GameTradingSystem/src/main/java/com/zxy/ps5gametradingsystem/entity/Game.java
package com.zxy.ps5gametradingsystem.entity;

import com.baomidou.mybatisplus.annotation.TableName;
import lombok.Data;

@Data
@TableName("`game`")
public class Game {
    private  String id;
    private  String englishName;
    private  String gameName;
    private  Integer price;
    private  Float discount;
    private  Integer inventory;
    private  String category;
    private  Integer sales;
    private  String introduction;
    private  Integer year;
    private  String img;
}

File: vue/ps5GameTradingSystem/src/types/game.ts
import type {favoriteItem} from "@/types/favorite.ts";

export interface game{
    id:string,
    englishName:string,
    gameName:string,
    price:number,
    discount:number,
    inventory:number,
    category:string
    sales:number,
    introduction:string,
    year:string,
    img:string,
}
export interface gameItemMap {
    id:string,
    englishName:string,
    gameName:string,
    price:number,
    discount:number,
    inventory:number,
    category:string
    sales:number,
    introduction:string,
    year:string,
    img:string,
    checked: boolean  // 前端专属的选中状态
}
export type games=game[]
export type gameItemMaps = gameItemMap[];